.github/workflows/changelog.yml

```diff
@@ -9,6 +9,7 @@ on:
     types: [opened, synchronize, reopened, labeled, unlabeled]
     branches:
       - main
+      - next
```

Comment:
Eventually we'll be able to drop this change, right?

Reply:
Yes - I'll follow up with a PR to clean this up :)

Reply:
I think we likely could drop it now. The `next` branch will go away

Reply:
@pichlermarc when you get around to this, PRs to the v1.x maintenance branch no longer triggers this workflow. As the two branches diverge over time, there would probably be _some_ PRs that needs to target that branch once in a while. I would fix it, but I can never remember whether you are supposed to change the workflow file on main or the one on the branch :)